docs/upgrade.md

```diff
@@ -4,3 +4,5 @@
 - In rollingDeploy each node is update one by one, and incase any of the node goes in pending/crashing state during update the operator halts the update and does not update the other nodes. This requires manual intervation.
 - Default updates and cluster creation is in parallel. 
 - Operator supports ```HPA autosaling/v1beta1``` Spec in the nodeSpec for druid nodes. In case HPA deployed, HPA controller maintains the replica count/state for the particular statefulset referenced.  Refer to ```examples.md``` for HPA configuration. Prefered to scale only brokers using HPA.
+- During upgrade if sts is set to ordered ready, the sts controller will not recover from crashloopback state. The issues is referenced [here](https://github.com/kubernetes/kubernetes/issues/67250), and here's a reference [doc](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#forced-rollback)
+- How operator solves this is using the ```forcedRollbackSts``` key, if this key is set to ```true``` the operator will delete the sts pod if its in crashloopback state. Example Scenario: During upgrade, user rolls out a faulty configuration causing the historical pod going in crashing state, user rolls out a valid configuration, the new configuration will not be applied unless user manual delete pods, so solve this scenario operator shall delete the pod automatically without user intervention. ```NOTE: User must be aware of this feature, their might be cases where crashloopback might be caused due probe failure, fault image etc, the operator shall keep on deleting on each re-concile loop```
```

Comment:
s/their might be cases/there might be cases/g
Can you also mention the default value of this new flag in the docs ?

Reply:
sure will do